<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-color: black;
	background-image:url(\&quot;background.png\&quot;);
};</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="simulationScene">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>295</width>
      <height>420</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>470</y>
      <width>66</width>
      <height>66</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{  
   color: rgb(255, 255, 255);
	border-image:url(\&quot;play.png\&quot;);
   border-radius:45px;
   background:transparent;
}

QPushButton:pressed{
    border-width:2px; 
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QTextBrowser" name="messageScene">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>311</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: black;</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="timeScene">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>380</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: black;
color:red;
border-color:white;
border-width:3px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>470</y>
      <width>66</width>
      <height>66</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{  
   color: rgb(255, 255, 255);
	border-image:url(\&quot;pause.png\&quot;);
   border-radius:45px;
   background:transparent;
}

QPushButton:pressed{
    border-width:2px; 
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>470</y>
      <width>66</width>
      <height>66</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{  
   color: rgb(255, 255, 255);
	border-image:url(\&quot;stop.png\&quot;);
   border-radius:45px;
   min-width:66px;
   background:transparent;
}

QPushButton:pressed{
    border-width:2px; 
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="ipLineEdit">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>310</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: black;
color:white;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="portLineEdit">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>340</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: black;
color:white;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>310</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{  
   color: rgb(255, 255, 255);
	border-image:url(\&quot;button.png\&quot;);
   border-radius:45px;
   background:transparent;
	font-size:15px;

}

QPushButton:pressed{
    border-width:2px; 
}</string>
    </property>
    <property name="text">
     <string>CONNECT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>310</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>Ip Address:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>340</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
   <widget class="QSlider" name="volumeSlider">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>10</y>
      <width>16</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::handle:verticall {
       image: url(\&quot;button.png\&quot;);
};</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="muteButton">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>290</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{  
   color: rgb(255, 255, 255);
   border-radius:45px;
   background:transparent;
}

QPushButton:checked{
    border-width:2px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGraphicsView" name="pathDrawingScene">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>295</width>
      <height>420</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="startPathButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>440</y>
      <width>125</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{  
   color: rgb(255, 255, 255);
	border-image:url(\&quot;button.png\&quot;);
   border-radius:45px;
   background:transparent;
	font-size:15px;

}

QPushButton:pressed{
    border-width:2px; 
}</string>
    </property>
    <property name="text">
     <string>START PATH</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearPathButton">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>440</y>
      <width>125</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{  
   color: rgb(255, 255, 255);
	border-image:url(\&quot;button.png\&quot;);
   border-radius:45px;
   background:transparent;
	font-size:15px;

}

QPushButton:pressed{
    border-width:2px; 
}</string>
    </property>
    <property name="text">
     <string>CLEAR PATH</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
